ci-operator/step-registry/upi/libvirt/cleanup/post/upi-libvirt-cleanup-post-ref.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "upi/libvirt/cleanup/post/upi-libvirt-cleanup-post-ref.yaml",
+	"owners": {
+		"approvers": [
+			"deepsm007",
```

Comment:
As much as I appreciate our friend Deep, we should probably remove him from the default approval list since he's not on our team anymore and we don't want him auto-assigned our PRs.